### PR TITLE
VS Code themes that match VS Light and Dark, including semantic tokens

### PR DESCRIPTION
**Pull request by Colengms**
Attempts to more closely match VS (2017) themes.
Does not yet include new VS 2019 changes to Light and Dark.

### CODE REVIEW

File: Themes/package.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
+{
+  "name": "cpptools-themes",
+  "displayName": "C/C++ Themes",
+  "description": "UI Themes for C/C++ extension.",
+  "version": "0.24.1",
```
Comment (bobbrow) [inline Themes/package.json]: Update the version.

File: Themes/package.json [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  "publisher": "ms-vscode",
+  "preview": true,
```
Comment (bobbrow) [inline Themes/package.json]: Does this token tell VS Code to preview the theme when the extension is installed?  If you set this to false, can we bundle it with our extension after all?

**Review by bobbrow — CHANGES_REQUESTED** [review]

File: Themes/package.json (hunk 2 above)
Comment (Colengms) [reply Themes/package.json]: This does not appear to be related to themes.

https://code.visualstudio.com/api/references/extension-manifest


preview |   | boolean | Sets the extension to be flagged as a Preview in the Marketplace.
-- | -- | -- | --

In emails with @aeschli , he seemed to imply this behavior is not currently optional.

File: Themes/package.json (hunk 1 above)
Comment (Colengms) [reply Themes/package.json]: Should this have its own version number?  These would be in their own vsix, and we probably don't want to have to update its version every time we update cpptools.

Comment (bobbrow) [reply Themes/package.json]: Sorry, that's what I meant.  This would be version 1.0.0 probably.

File: Themes/package.json (hunk 2 above)
Comment (bobbrow) [reply Themes/package.json]: Ok.  Then we should set this to false since the theme extension will not be marked as "Preview" in the Marketplace.

**Review by bobbrow — APPROVED** [review]

**Pull request merged**